### PR TITLE
Fixed spelling error in comment in settings.

### PR DESCRIPTION
**Pull request by dsjen**
@dan-f -- I noticed a spelling error in the settings file.

### CODE REVIEW

**Comment by dan-f** [comment]
Haha nice one!

**Comment by dan-f** [comment]
:+1:

File: analytics_dashboard/settings/base.py [hunk 1]
```diff
@@ -318,7 +318,7 @@
 # used to construct the shortcut link to course modules
 LMS_COURSE_SHORTCUT_BASE_URL = None
 
-# used to construct shortcusts to view/edit courses in Studio
+# used to construct shortcut to view/edit courses in Studio
```
Comment (dan-f) [inline analytics_dashboard/settings/base.py]: shortcuts?

Comment (dsjen) [inline analytics_dashboard/settings/base.py]: I know.  It's amazing.

Comment (dan-f) [inline analytics_dashboard/settings/base.py]: Should this word be plural or singular though? As it stands, the whole comment is missing an article.

**Comment by dsjen** [comment]
I updated the comment--let me know how it looks.  I'll rebase off of https://github.com/edx/edx-analytics-dashboard/pull/388 once that's merged.

**Comment by dan-f** [comment]
:tulip: :bouquet: :rose: :ship:

**Pull request merged**